lib/puppet/util/package/version/pip.rb

```diff
@@ -0,0 +1,230 @@
+module Puppet::Util::Package::Version
+  class Pip
+    def initialize(version)
```

Comment:
we should be consistent between Version Classes, apt has the `initialize` method private and expose only parse.